doc/source/visualizing/geographic_projections_and_transforms.rst

```diff
@@ -0,0 +1,86 @@
+.. _geographic_projections_and_transforms:
+
+Geographic Projections and Transforms
+=====================================
+
+Geographic data that is on a sphere can be visualized by projecting that data
+onto a representation of that sphere flattened into 2d space. There exist a
+number of projection types, which can be found in the `the cartopy
+documentation <https://scitools.org.uk/cartopy/docs/v0.15/crs/projections.html>`_.
+yt now supports these projection types for geographically loaded data.
+Underlying data is assumed to have an underlying projection type of `PlateCarree`. If
+your data is not of this form, feel free to open an issue or file a pull
+request on the yt github page for this feature.
+
+It should be noted that
+these projections are not the same as yt's ProjectionPlot. For more information
+on yt's projection plots, see :ref:`projection-types`.
+
+Installation
+^^^^^^^^^^^^
+
+In order to access the geographic projection functionality, you will need to have an
+installation of cartopy available on your machine.
+
+.. code-block:: bash
+
+    conda install cartopy
+
+Using Basic Projections
+^^^^^^^^^^^^^^^^^^^^^^^
+
+All of the transforms available in Cartopy v0.15 are accessible with this
+functionality.
+
+The next few examples will use the GEOS dataset. For details about loading this
+data, please see the geographic projections cookbook.
+
+If a geographic dataset is loaded without any defined projection the default
+option of `PlateCarree` will be displayed.
+
+.. code-block:: python
+
+    ds = yt.load_uniform_grid(data, sizes, 1.0, geometry=("geographic", dims),
+    bbox=bbox)
+    p = yt.SlicePlot(ds, "altitude", 'AIRDENS')
+
+If an option other than `PlateCarree` is desired, the plot projection type can
+be set manually with the `set_mpl_projection` function. This next code block
+sets the projection to a `Robinson` projection from the default `PlateCarree`.
+
+.. code-block:: python
+
+    p.set_mpl_projection('Robinson')
```

Comment:
Is it possible to do this in the instantiation of `SlicePlot` itself, with an argument?

Reply:
Right now I don't think so, but I could add that if it'd be useful.

Reply:
Ok, I've added in that functionality and updated the test and the documentation to reflect this additional functionality. We should now be able to pass in `geo_projection=`  into `SlicePlot` and the initial plot should have a projection other than `PlateCarree`.